### PR TITLE
Creating of current forecast view

### PR DESCRIPTION
**Pull request by glebdronchenko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "npm": ">=3"
   },
   "dependencies": {
-    "axios": "^0.21.4",
+    "axios": "0.21.4",
+    "chart.js": "^3.5.1",
```
Comment (anton-vlasik) [inline package.json:23]: you need both chart.js and react-chartjs-2 ?

File: src/components/CurrentCityForecast/CurrentCityForecast.module.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
+@import '../../assets/styles/constants.scss';
+
+$current-temperature-font-size: 95px;
+$temperature-type-content: '℃';
```
Comment (anton-vlasik) [inline src/components/CurrentCityForecast/CurrentCityForecast.module.scss]: you also have something called degrees-content somewhere else? why?

File: src/components/CurrentCityForecast/components/HourlyForecastChart/HourlyForecastChart.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import { Line } from 'react-chartjs-2';
+import hourlyForecastData from './mockData';
+
+function HourlyForecastChart() {
+  const { hours, temperatures } = prepareChartData(hourlyForecastData);
```
Comment (anton-vlasik) [inline src/components/CurrentCityForecast/components/HourlyForecastChart/HourlyForecastChart.jsx:7]: It seems like all that is happening here inside this function have nothing to do with this component's data and totally static AND should be put outside of it. Everything from line 5 to line 45 can be moved to utilities folder

File: package.json (hunk 1 above)
Comment (glebdronchenko) [reply package.json:23]: As I understand react-chartjs-2 is just react wrapper for chart.js.

File: src/components/CurrentCityForecast/CurrentCityForecast.module.scss (hunk 2 above)
Comment (glebdronchenko) [reply src/components/CurrentCityForecast/CurrentCityForecast.module.scss]: Done. I have moved this variable to constants.

File: src/components/CurrentCityForecast/components/HourlyForecastChart/HourlyForecastChart.jsx (hunk 3 above)
Comment (glebdronchenko) [reply src/components/CurrentCityForecast/components/HourlyForecastChart/HourlyForecastChart.jsx:7]: I agree with your remarks. Done.

File: package.json (hunk 1 above)
Comment (glebdronchenko) [reply package.json:23]: This lib author wrote here https://www.npmjs.com/package/react-chartjs-2 about it.

File: src/components/CurrentCityForecast/CurrentCityForecast.jsx [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+import HourlyForecastChart from './components/HourlyForecastChart/HourlyForecastChart';
+
 function CurrentCityForecast() {
-  return <div>CurrentCityForecast</div>;
+  return (
+    <div className={classes.currentCityContainer}>
+      <div className={classes.currentCityInfo}>
+        <div className={classes.forecastInfo}>
+          <div className={classes.mainInfo}>
+            <img
```
Comment (DmitriyKrivovyaz) [inline src/components/CurrentCityForecast/CurrentCityForecast.jsx:12]: need to add alt to your img tag

File: src/components/CurrentCityForecast/components/DailyForecasts/DailyForecast/DailyForecast.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import classes from './DailyForecast.module.scss';
+
+function DailyForecast() {
+  return (
+    <div className={classes.forecastItem}>
+      <div className={classes.title}>SU</div>
+      <img
```
Comment (DmitriyKrivovyaz) [inline src/components/CurrentCityForecast/components/DailyForecasts/DailyForecast/DailyForecast.jsx:7]: add alt

File: src/components/CurrentCityForecast/CurrentCityForecast.jsx (hunk 4 above)
Comment (glebdronchenko) [reply src/components/CurrentCityForecast/CurrentCityForecast.jsx:12]: Done.

File: src/components/CurrentCityForecast/components/DailyForecasts/DailyForecast/DailyForecast.jsx (hunk 5 above)
Comment (glebdronchenko) [reply src/components/CurrentCityForecast/components/DailyForecasts/DailyForecast/DailyForecast.jsx:7]: Done.

**Review by AV29 — APPROVED** [review]

File: src/components/CurrentCityForecast/components/DailyForecasts/DailyForecasts.jsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+function DailyForecasts() {
+  return (
+    <div className={classes.forecastsContainer}>
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
```
Comment (alvikalvik) [inline src/components/CurrentCityForecast/components/DailyForecasts/DailyForecasts.jsx:14]: Will you replace this to a loop in a future?

Comment (glebdronchenko) [reply src/components/CurrentCityForecast/components/DailyForecasts/DailyForecasts.jsx:14]: Yes. I will do it, when I will connect this component with API.

**Review by ximet — APPROVED** [review]

**Pull request merged**